### ISSUE TITLE
About Label

### ISSUE THREAD

**Issue by 870572761** [Linfeng-Tang/SeAFusion#6]
Why the dataset's labels are all black?

**Comment by Linfeng-Tang** [comment]
In the original labels, 0, 1, 2, 3, ... , 8 are used to represent the different categories, so all the labels are black. You can write a mapping code to visualize the labels.

**Comment by 870572761** [comment]
> In the original labels, 0, 1, 2, 3, ... , 8 are used to represent the different categories, so all the labels are black. You can write a mapping code to visualize the labels.

Thanks.

**Comment by 870572761** [comment]
I find some labels just exist one type. Is it a special label or my wrong?`
There is my code.
 `import numpy as np
from PIL import Image
import torch
label_path1='.\leir\label\js.jpg'
a=Image.open(label_path1)
label = np.array(a)

label = np.asarray(Image.fromarray(label), dtype=np.int64)

lb=torch.tensor(label)
lb = np.uint8(255.0 * lb)

lb = Image.fromarray(lb)
lb.show()
`

**Comment by Linfeng-Tang** [comment]
Apologies for the delay in responding. Have you resolved the previous issues or do you still need assistance? If you still need help, I can share the mapping code I have recently written with you.

**Comment by 870572761** [comment]
> Apologies for the delay in responding. Have you resolved the previous issues or do you still need assistance? If you still need help, I can share the mapping code I have recently written with y
Thanks.I really need that.It is my email: l870572761@gmail.com.

**Comment by 870572761** [comment]
> Apologies for the delay in responding. Have you resolved the previous issues or do you still need assistance? If you still need help, I can share the mapping code I have recently written with you.

谢谢作者，我非常需要这个映射代码，我的邮箱是：l870572761@gmail.com
